ebos/ecltransmissibility_impl.hh

```diff
@@ -157,6 +170,7 @@ update(bool global, const std::function<unsigned int(unsigned int)>& map, const
     // get the ntg values, the ntg values are modified for the cells merged with minpv
     const std::vector<double>& ntg = eclState_.fieldProps().get_double("NTG");
     const bool updateDiffusivity = eclState_.getSimulationConfig().isDiffusive() && enableDiffusivity_;
+    const bool updateDispersivity = eclState_.getSimulationConfig().rock_config().dispersion() && enableDispersivity_;
     unsigned numElements = elemMapper.size();
```

Comment:
Is enabledispersivity given in the deck or a compile time option. If it is a compile time option I think the run should fail.

Reply:
It is a compile time option. But since the dispersivity returns 0.0 if empty (as in diffusivity), it does not fail.

Reply:
If the rock_config().dispersion() is true and enableDispersivity_ is false we should abort the run, as we are trying to run a deck with dispersion in a simulator instantiation that does not honor it. Same with diffusion.